GLTFSerialization/GLTFSerialization/Schema/GLTFRoot.cs

```diff
@@ -242,6 +242,11 @@ public GLTFRoot(GLTFRoot gltfRoot) : base(gltfRoot)
 			}
 		}
 
+		/// <summary>
+		/// Whether this object is a GLB
+		/// </summary>
+		public bool IsGLB;
```

Comment:
This seems to be unused

Reply:
The parser sets it and it's an externally used properly